projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas-submit-button/go-off-canvas-submit-button.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { GoOffCanvasSubmitButtonComponent } from './go-off-canvas-submit-button.component';
+
+describe('GoOffCanvasSubmitButtonComponent', () => {
+  let component: GoOffCanvasSubmitButtonComponent;
+  let fixture: ComponentFixture<GoOffCanvasSubmitButtonComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ GoOffCanvasSubmitButtonComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(GoOffCanvasSubmitButtonComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Should we add a test here to test that `action` is called (or emitted if we decided to change that) when the button is clicked?